src/Server/Remote/NativeSsh.php

```diff
@@ -49,7 +50,6 @@ public function run($command)
         $serverConfig = $this->getConfiguration();
         $sshOptions = [
             '-A',
-            '-q',
```

Comment:
@elfet Is there a real reason of adding "-q" option? Without it `ssh` returns 255 status code (Unknown error) and empty output.